.github/workflows/update_schema.yaml

```diff
@@ -0,0 +1,43 @@
+name: Deploy Liquibase Schema
+
+on:
+  workflow_dispatch:
+    branches:
+      - main
+
+jobs:
+  deploy-schema:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+
+    - name: Set up Java
+      uses: actions/setup-java@v3
+      with:
+        distribution: "temurin"
+        java-version: "17"
+
+    - name: Download Liquibase
+      run: |
+        curl -L https://github.com/liquibase/liquibase/releases/download/v4.26.0/liquibase-4.26.0.zip -o liquibase.zip
+        unzip liquibase.zip -d liquibase
+
+    - name: Download MariaDB JDBC Driver
+      run: |
+        wget https://repo1.maven.org/maven2/org/mariadb/jdbc/mariadb-java-client/3.0.3/mariadb-java-client-3.0.3.jar
+
+    - name: Run Liquibase Migration
+      env:
+        DATABASE_URL: jdbc:mariadb://${{ secrets.DATABASE_URL }}
```

Comment:
I thought our database won't allow connections like this without ssh tunelling, and if it does we should block it.
I had considered cloud run job doing the actual schema migration, and this action triggering that with the required project.xml file.

We should also have some way to target the production / development database.

Reply:
Yes! You're totally right. I've updated the `yaml` files and made the `Dockerfile` needed to build the Cloud Run job. The code for the job should be in the python file however. One of the steps in the updated github action is to build and push the docker image to a registry. I haven't explored our docker registry so that is probably I step I would need assistance with so that we are pushing to the right registry.